specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,10340 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "MicrosoftScanningClient",
+    "description": "Creates a Microsoft.Scanning management client.",
+    "version": "2023-09-01"
+  },
+  "host": "purview.azure.com",
+  "basePath": "/scan",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "flow": "implicit",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      },
+      "description": "Azure Active Directory OAuth2 Flow."
+    }
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/endpoint"
+      }
+    ]
+  },
+  "paths": {
+    "/azureKeyVaults/{azureKeyVaultName}": {
+      "get": {
+        "tags": [
+          "AzureKeyVaults"
+        ],
+        "description": "Gets azureKeyVault connection.",
+        "operationId": "KeyVaultConnections_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "azureKeyVaultName",
+            "required": true,
+            "type": "string"
+          },
```

Comment:
Please add descriptions on all parameters, especially path parameters.

Reply:
Added a series of parameters including this one here